src/client/dfuse/il/int_posix.c

```diff
@@ -802,7 +802,9 @@ child_hdlr(void)
 {
 	int rc;
 
-	daos_eq_lib_reset_after_fork();
+	rc = daos_eq_lib_reset_after_fork();
+	if (rc)
+		DFUSE_LOG_WARNING("daos_eq_lib_init() failed in child process: " DF_RC, DP_RC(rc));
```

Comment:
Nit, we're trying to move away from these older dfuse specific logging macros.
```suggestion
	if (rc)
		DL_WARN(rc, "daos_eq_lib_init() failed in child process");
```

Reply:
Fixed as suggested. Thank you!